tests/components/huisbaasje/test_sensor.py

```diff
@@ -41,25 +60,255 @@ async def test_setup_entry(hass: HomeAssistant):
         await hass.async_block_till_done()
 
         # Assert data is loaded
-        assert hass.states.get("sensor.huisbaasje_current_power").state == "1012.0"
-        assert hass.states.get("sensor.huisbaasje_current_power_in").state == "1012.0"
+        current_power = hass.states.get("sensor.huisbaasje_current_power")
+        assert current_power.state == "1012.0"
+        assert current_power.attributes.get(ATTR_DEVICE_CLASS) == DEVICE_CLASS_POWER
+        assert current_power.attributes.get(ATTR_ICON) == "mdi:lightning-bolt"
```

Comment:
For a future PR, having an icon and a device class is actually incorrect. When using a device class, an icon should not be defined.

However, this is already an existing issue, so ignoring it for now.